addon/components/denali-textarea.hbs

```diff
@@ -0,0 +1,10 @@
+{{! Copyright 2020, Verizon Media. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<div class="input
+  {{this.activeClass}}
+  {{this.warningClass}}"
+>
+  <textarea ...attributes>{{@value}}</textarea>
```

Comment:
```suggestion
  <textarea ...attributes>{{yield}}</textarea>
```

Reply:
Just changing @value to yield causes the tests on values to fail. Do you have other changes you would suggest to resolve that? or should I keep it as @value?

Reply:
My suggestion is to make your own pr fix commit and then push that once all the tests are complete and passing. Don't bother with accepting individual commit suggestions here on github unless they are trivial.

Reply:
You gotta test the yield with a snippet like this.

```hbs

<DenaliTextArea>
  Passed Yield Value
</DenaliTextArea>

```